### DISCUSSION TITLE
Welcome to Joalor64 Engine Rewrite Discussions!

### DISCUSSION THREAD

**Discussion by Joalor64** [Joalor64/Joalor64-Engine-Rewrite#28] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by BambiGaming2022** [comment]
Is there any extra keys?

**Reply by moxie-coder** [reply]
nope, not yet
